packages/compass-sidebar/src/stores/store.js

```diff
@@ -125,7 +125,7 @@ store.onActivated = (appRegistry) => {
     store.dispatch(changeActiveNamespace(dbName));
   });
 
-  appRegistry.on('select-instance', () => {
+  appRegistry.on('sidebar-instance-workspace-open-tab', () => {
```

Comment:
super non-blocking .. what does this name stand for? Is it more something like `workspace-open-tab-from-sidebar`? Why it was `select-instance` before?

Reply:
It emits from the sidebar, so I put the sidebar in the event name as the first thing similar to how I done it [in the databases navigation section](https://github.com/mongodb-js/compass/blob/199d42553a2b3f8af37bed621476952f86847a5f/packages/compass-sidebar/src/components/sidebar-databases-navigation/index.js#L58-L66). Should I change it to something else there too while I'm at it? I can also keep it as `select-instance` if you feel like it's better

Reply:
Just to be clear I think we can go on with `sidebar-instance-workspace-open-tab` too.

`select-instance` maybe at this point is a bit too generic, at least I would not understand what it does now. What does "instance" stands for here?

A convention where we add the source in the event could be a great idea, but at the moment is not very clear, it just seems like any other word since everything is separated by `-` and that is not something happening for all the events. Would be different if the first separator is different somehow: `sidebar/instance-workspace-open-tab`.

How would you describe this action in plain English?

Reply:
So `instance` is how Compass refers to the instance you are connected to (see `instance-model` or `Instance.Workspace`), I agree that it's maybe not the best name, but I'm not inventing new terms here, just trying to keep it consistent with other parts of the app, I added workspace to the name so it's a bit clearer that we are not actually selecting a different instance.

FWIW you would usually use `:` as a delimiter and compass even does it sometimes (but not everywhere and I honestly struggle to find a pattern, seems to be mostly done in telemetry? 🤷 ) and I'm just not sure what to use as there is no conventions anywhere and mixing two in one place (other events emitted from sidebar don't have any delimiter) felt weird.

Probably something like `open instance workspace with tab n selected (optionally)`, so the event might be `open-instance-workspace` and tabName as an argument? Makes sense?

Reply:
Renamed it to `open-instance-workspace` and will merge now, but please tell me if you want me to change the event name to something else